File: Database/Entities/Sprint.cs
using Database.Entities.Base;

namespace Database;

public class Sprint : BaseEntity
{
    public DateTime? StartDate { get; set; }
    public DateTime? EndDate { get; set; }
    public string? Status { get; set; }
    public string ProjectId { get;set; }
    public string Title { get;set; }
}

File: TrackEdize/ClientApp/src/app/tracking/models/sprint.ts
import { Dropdown } from "src/app/shared/model/dropdown";
import {Comment} from './comment';

export class Sprint {
    id?: string;
    createdDate?: Date;
    updatedDate?: Date;
    startDate: Date | undefined;
    endDate: Date | undefined;
    title: string | null = null;
    description: string | null = null;
    projectId: string | null = null;
    status: string | null = null;

    constructor() {
        
    }
}